src/en/credentials.md

```diff
@@ -417,18 +425,31 @@ Credentials:
 
 ### Removing local credentials
 
-If a local credential (i.e. not cached on a controller) is no longer required,
-it can be removed:
+The `remove-credential` command is used to remove a local credential (i.e. not
+cached on a controller):
 
 ```bash
 juju remove-credential aws bob
 ```
 
+### Setting a remote credential for a model
+
+To set what remote credential should be used for a model the `set-credential`
+command (`v.2.5.0`) is available. For instance, to have remote credential 'bob'
+be used for model 'trinity' (for cloud 'aws'):
+
+```bash
+juju set-credential -m trinity aws bob
+```
+
+!!! Note:
```

Comment:
Maybe worthwhile mentioning that the user has to be either a controller admin or a model admin in order to be able to replace model's credential?

Reply:
Alright.